### ISSUE TITLE
[Desktop UI] Increase size of data ticker to accommodate all languages

### ISSUE THREAD

**Issue by carsonefv** [getlantern/lantern#5400]
Currently the text on multiple languages is too long for the space provide underneath the data ticker:

Vietnamese:
![screenshot 2016-11-04 11 15 36](https://cloud.githubusercontent.com/assets/4400763/20010946/5ddaf710-a280-11e6-8c0e-f5c46c0d1800.png)
French:
![screenshot 2016-11-04 11 15 51](https://cloud.githubusercontent.com/assets/4400763/20010951/634c3c36-a280-11e6-98c8-826578cfe6f8.png)

We should extend the size of this section to ensure the text in any language will fit correctly as follows:
Data ticker bar length: 280px
Text Box description length: 280px
Data Ticker Header Shape length: 313px
![screenshot 2016-11-04 11 29 37](https://cloud.githubusercontent.com/assets/4400763/20011426/007368da-a282-11e6-8c49-31326802beaa.png)

**Comment by carsonefv** [comment]
@myleshorton can we get this into the next sprint?

**Comment by myleshorton** [comment]
Yeah sure @carsonefv. @joesis if you feel like jumping on this any time, just go for it.

**Comment by joesis** [comment]
Sure will do
